src/main/java/org/usfirst/frc4904/robot/commands/HatchIntake.java

```diff
@@ -0,0 +1,12 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.FloorIO;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+
+public class HatchIntake extends MotorConstant {
```

Comment:
Rename, since this command doesn't intake the hatch, it simply rolls it into position to be grabbed by the velcro.